firefox_ui_tests/remote/security/test_ssl_disabled_error_page.py

```diff
@@ -0,0 +1,96 @@
+        # also: utils.sanitize used by 3 geolocation tests; better to add to library, maybe prefs?
+        # also: https://dxr.mozilla.org/mozilla-central/source/browser/base/content/sanitize.js#417
+        try:
+            self.marionette.execute_async_script("""
+                Cu.import("resource://gre/modules/Services.jsm");
+
+                var spec = { sessions: true };
+
+                // Load the sanitize script
+                var tempScope = {};
+                Cc["@mozilla.org/moz/jssubscript-loader;1"]
+                .getService(Ci.mozIJSSubScriptLoader)
+                .loadSubScript("chrome://browser/content/sanitize.js", tempScope);
+
+                // Instantiate the Sanitizer
+                var s = new tempScope.Sanitizer();
+                s.prefDomain = "privacy.cpd.";
+                var itemPrefs = Services.prefs.getBranch(s.prefDomain);
+
+                // Apply options for what to sanitize
+                for (var pref in spec) {
+                  itemPrefs.setBoolPref(pref, spec[pref]);
+                };
+
+                // Sanitize and wait for the promise to resolve
+                var finished = false;
+                s.sanitize().then(() => {
+                  for (let pref in spec) {
+                    itemPrefs.clearUserPref(pref);
+                  };
+                  marionetteScriptFinished(true);
+                }, aError => {
+                  for (let pref in spec) {
+                    itemPrefs.clearUserPref(pref);
+                  };
+                  marionetteScriptFinished(false);
+                });
+            """, script_timeout=10000)
+        except TimeoutException:
+            # TODO?
+            pass
+
+        # Disable SSL 3.0, TLS 1.0 and TLS 1.1 for secure connections
+        # by forcing the use of TLS 1.2
+        # see: http://kb.mozillazine.org/Security.tls.version.*#Possible_values_and_their_effects
+        self.prefs.set_pref('security.tls.version.min', 3)
+        self.prefs.set_pref('security.tls.version.max', 3)
+
+    def tearDown(self):
+        try:
+            self.prefs.restore_all_prefs()
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    def test_ssl_disabled_error_page(self):
+        with self.marionette.using_context('content'):
+            # Open the test page
+            self.assertRaises(MarionetteException, self.marionette.navigate, self.url)
+
+            # Wait for the DOM to receive events
+            time.sleep(1)
+
+            # Verify "Secure Connection Failed" error page title
+            title = self.marionette.find_element(By.ID, 'errorTitleText')
+
+            nss_failure2title = self.browser.get_entity('nssFailure2.title')
+            self.assertEquals(title.get_attribute('textContent'), nss_failure2title)
+
+            # Verify "Try Again" button appears
+            self.assertIsNotNone(self.marionette.find_element(By.ID, 'errorTryAgain'))
```

Comment:
Maybe better to check for `is_displayed` here. It should work given that it is content scope, right?